CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Changes in September 2021
 
+-   Within Core Import Export, added `definitionName` parameter to
```

Comment:
Sorry about being a pain but for consistency we are trying to limit the starting word so that we can group the additions, updates and removals in the change log. Please see https://github.com/constellation-app/constellation/blob/master/CHECKLIST.md and mid way down you will see the start words.